tests/ci/docker_images/linux-x86/push_images.sh

```diff
@@ -53,6 +53,16 @@ docker tag ubuntu-19.10:sanitizer ${ECS_REPO}:ubuntu-19.10_clang-9x_sanitizer_la
 docker push ${ECS_REPO}:ubuntu-19.10_clang-9x_sanitizer_latest
 docker push ${ECS_REPO}:ubuntu-19.10_clang-9x_sanitizer_`date +%Y-%m-%d`
 
+docker tag ubuntu-20.04:gcc-9x ${ECS_REPO}:ubuntu-20.04_gcc-9x_`date +%Y-%m-%d`
+docker tag ubuntu-20.04:gcc-9x ${ECS_REPO}:ubuntu-20.04_gcc-9x_latest
```

Comment:
Nit: I put `ubuntu-20.04` at the end because 19.10 has sanitizer dimension that other image does not support.

Reply:
IMO better to group according to OS.